test/unit/_suite.ts

```diff
@@ -11,7 +11,7 @@ export interface Utils {
 export default suite<Utils, any>((tests) => {
   let sandbox: sinon.SinonSandbox;
 
-  beforeEach(() => sandbox = sinon.sandbox.create());
+  beforeEach(() => sandbox = sinon.createSandbox());
```

Comment:
This works, but we no longer need to create a sandbox because `sinon` is now the default sandbox. That means we can remove this `beforeEach` and do `sinon.restore()` in the `afterEach`. The `spy` and `stub` definitions below can be simplified to:
```js
spy: sandbox.spy,
stub: sandbox.stub
```